packages/react-i18n/src/constants/index.ts

```diff
@@ -38,6 +38,18 @@ export enum Weekday {
   Saturday = 'saturday',
 }
 
+export const undefinedDateDisplayProps = {
+  era: undefined,
```

Comment:
I don't think that it's a good idea to maintain a list of properties to wipe. We'd have to keep updating this as we add support for additional options.

A better fix could be to pluck the options that we allow in that context, instead of spreading the entire options object.

But, I think the root cause of this issue could be something else: I suspect something to do with an unexpected DateStyle being passed in.

Reply:
passing in options still works (see my tests). I only use this when the functions isolate certain things like `monthDay` and `time`. right now, these aren't working properly. For example if I pass in `timeZoneName` as an option, `monthDay` will display the time zone.

Reply:
`humanize` splits up the date and the time so it can send it to a translation i.e. `{date} at {time}`. Right now the functions aren't splitting them up properly when options are passed in. I need to clear the options and only specify the properties that are associated with `date` and `time` separately. So I pass the options through, but only the `time` props to the time consts and only the `date` props to the date consts